azure-pipelines.yml

```diff
@@ -137,3 +137,12 @@ jobs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)'
       ArtifactName: 'drop'
       publishLocation: 'Container'
+  - task: NuGetCommand@2
+    condition: eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], 'True')
+    inputs:
+      command: 'push'
+      packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
+      nuGetFeedType: 'internal'
+      publishVstsFeed: 'e6a70c92-4128-439f-8012-382fe78d6396/f37f760c-aebd-443e-9714-ce725cd427df'
```

Comment:
what is the link to the feed (just curious)?

Reply:
The package will be upload it to `https://azfunc.visualstudio.com/Azure%20Functions/_packaging?_a=feed&feed=AzureFunctionsPreRelease`. If no changes have been done to the language worker code base, then the package version will not be updated. Also, the tasks is configure to skip duplicates as shown below.

![image](https://user-images.githubusercontent.com/12720858/119744331-9dd37f00-be40-11eb-894c-94efea463bcc.png)